notificationchannel/store.go

```diff
@@ -16,6 +17,7 @@ import (
 type Store interface {
 	FindAll(context.Context) ([]Channel, error)
 	FindOne(context.Context, uuid.UUID) (*Channel, error)
+	FindMany(context.Context, []string) ([]Channel, error)
```

Comment:
Should this take an array of uuids instead, or is string sufficient?

Reply:
I think string is okay for now, our validation and `sqlutil` stuff expects strings when dealing with multiple (same as `DeleteManyTx`) eventually we should bring everything to UUID, but for now I followed the existing convention in this package to keep it focused on the fix until we can refactor the rest.